contrib/pylightning/lightning/lightning.py

```diff
@@ -22,20 +27,48 @@ class Millisatoshi:
 
     Many JSON API fields are expressed in millisatoshis: these automatically get
     turned into Millisatoshi types.  Converts to and from int.
+
+    For common Bitcoin currency units see: https://en.bitcoin.it/wiki/Units
     """
     def __init__(self, v):
         """
-        Takes either a string ending in 'msat', 'sat', 'btc' or an integer.
+        Takes either a string ending in 'msat', 'sat', 'btc', "mbtc", "finney", ... or an integer.
```

Comment:
nit: maybe choose between `"` or `'` for consistency ?

Reply:
:D will do